themes/jquery/css/base.css

```
@@ -3793,6 +3793,8 @@ pre .diff .hljs-comment {
 .syntaxhighlighter table td.gutter .line {
 	text-align: right !important;
 	padding: 0 0.5em 0 1em !important;
+	user-select: none;
+	-webkit-user-select: none;
```

Please declare the vendor-prefixed version first to ensure the unprefixed one takes priority when both are supported; let's also add `-ms-user-select` for IE:
```suggestion
	-webkit-user-select: none;
	-ms-user-select: none;
	user-select: none;
```

---

Oh, good call(s), thanks! :+1: